cypress/support/pages/user.PageObject.js

```diff
@@ -0,0 +1,33 @@
+import PageObject from "../PageObject";
+
+class UserPageObject extends PageObject {
+  visitUserPage(username) {
+    cy.visit(`/#/@${username}`);
+  }
+
+  get followBtn() {
+    return cy.getByDataCy('follow-unfollow-btn');
```

Comment:
The `followBtn` and `unfollowBtn` getters both use the same `data-cy` attribute ('follow-unfollow-btn'). Ensure that the application logic correctly distinguishes between the follow and unfollow states, possibly by checking the button text or another attribute.